libraries/ESP8266mDNS/src/LEAmDNS.cpp

```diff
@@ -767,6 +767,10 @@ uint32_t MDNSResponder::queryService(const char* p_pcService,
                                      const char* p_pcProtocol,
                                      const uint16_t p_u16Timeout /*= MDNS_QUERYSERVICES_WAIT_TIME*/)
 {
+    if (0 == m_pUDPContext)
```

Comment:
Isn't this a ptr? check against nullptr?

Reply:
Yes. For coherency I sticked to this file's style.
https://github.com/esp8266/Arduino/blob/e1789ddf0c73ff00cdca104e420ead5bc8bc016f/libraries/ESP8266mDNS/src/LEAmDNS.cpp#L101

I was afraid to modify too much for when LEAmDNS2 is coming to avoid merge complication.